source/Assets/Plugins/iOS/TwitterKitIOSWrapper.mm

```diff
@@ -26,6 +26,7 @@
 static const char * TWTRUnityAPIMethodTweetComplete = "TweetComplete";
 static const char * TWTRUnityAPIMethodTweetFailed = "TweetFailed";
 static const char * TWTRUnityAPIMethodTweetCancelled = "TweetCancelled";
+UIViewController * UnityGetGLViewController();
```

Comment:
Looks like we don't need this declaration anymore?
http://answers.unity3d.com/comments/1100699/view.html
I tested the project w/o it and the composer was presented successfully.

Reply:
Right- I've always forward-declared it in my plugins, should I want to compile a static library. But since you all include the .mm as part of the Unity project, that line is perhaps unnecessary.